pymess/migrations/0026_migration.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.1 on 2021-12-31 10:25
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('pymess', '0025_migration'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='pushnotificationtemplate',
+            name='redirect_url',
+            field=models.TextField(blank=True, null=True),
```

Comment:
you created migration from core project. But we have turned of verbose name in the migrations. Add them by hand or turn off django_migrations_ignore_attrs in the EXTRA_APPS and generate migration again

Reply:
good to know. thx. Fixed.